website/docs/d/log_profile.html.markdown

```diff
@@ -0,0 +1,50 @@
+---
```

Comment:
This file should be added to the [data source ToC](https://github.com/terraform-providers/terraform-provider-azurerm/blob/master/website/azurerm.erb#L28).

Reply:
added